### ISSUE TITLE
ESI Hardware: Nodes marked as having Issues by ESI

### ISSUE THREAD

**Issue by hakasapl** [CCI-MOC/MOC-issues#197]
- [ ] MOC-R4PAC21U07-S3 - Node fails inspection
- [ ] MOC-R4PAC22U09-S3 - Node fails inspection
- [ ] MOC-R4PAC22U07-S1 - Node fails inspection
- [ ] MOC-R4PAC24U29-S3 - Node fails inspection
- [ ] MOC-R4PAC24U23-S1 - Node fails inspection
- [ ] MOC-R4PAC24U23-S3 - Node fails inspection
- [ ] MOC-R4PAC24U19-S1 - Node fails inspection
- [ ] MOC-R4PAC24U15-S1 - Node fails inspection
- [ ] MOC-R4PAC24U13-S1 - Node fails inspection
- [ ] MOC-R4PAC24U11-S1 - Node fails inspection
- [ ] MOC-R4PAC24U07-S3 - Node fails inspection
- [ ] MOC-R4PAC22U17-S3 - Node fails to provision
- [ ] MOC-R4PAC22U15-S3 - Node fails to provision
- [ ] MOC-R4PAC22U15-S1 - Node fails to provision
- [ ] MOC-R4PAC24U03-S3 - Node fails to provision
- [ ] MOC-R4PAC22U31-S1A - Node fails to provision
- [ ] MOC-R4PAC24U37-S3C - Node fails to provision
- [ ] MOC-R4PAC24U31-S1A - Node fails to provision
- [ ] MOC-R4PAC21U37-S3B - nic seems to not be cabled up      
- [ ] MOC-R4PAC21U37-S3C - ipxe initialising devices... ok then exits                                                                                               
- [ ] MOC-R4PAC21U35-S1A - some old pxe version?      
- [ ] MOC-R4PAC21U35-S1C - some old pxe version?      
- [ ] MOC-R4PAC21U35-S3A - some old pxe version? 
- [ ] MOC-R4PAC22U35-S3A - ipxe initialising devices... ok then exits
- [ ] MOC-R4PAC22U33-S1A - won't turn on, even from idrac (before: won't boot from hard drive after provisioning) 
- [ ] MOC-R4PAC21U35-S1B - provisioned, but can't ping; its baremetal port has mac a8:99:69:84:47:2f which is mapped to 1/13/2 in Ironic, but on the switch the mac is mapped to 1/13/3
- [ ] MOC-R4PAC21U35-S3B - provisioned, but can't ping; its baremetal port has mac a8:99:69:84:48:9d which is mapped to 1/14/2 in Ironic, but on the switch the mac is mapped to 1/13/1
- [ ] MOC-R4PAC10U33-S3A - won't boot, system says "System BIOS has halted."

**Comment by hakasapl** [comment]
@imstof any progress on this?

**Comment by imstof** [comment]
we'll get started on them now

**Comment by imstof** [comment]
@hakasapl what are ipmi credentials? it looked like it might be default dell root:calvin from the node show output, but that is not working.

**Comment by hakasapl** [comment]
@imstof just sent an email

**Comment by hakasapl** [comment]
@imstof let me know if you wanted to meet about anything here, sorry if the docs are not clear. Let me know any nodes that are back working as you work on them and I can move them back to prod

**Comment by hakasapl** [comment]
@imstof any updates on this?

**Comment by imstof** [comment]
@hakasapl  I've been flat-out at SC24 all week. I will update you Monday

**Comment by hakasapl** [comment]
These nodes are in various states of issue as listed. Generally it is best to watch console while inspecting through ESI to see whats going on - it could just be a boot order issue and it's booting off HDD instead. If it is a hardware issue feel free to swap it out and update manifests.

**Comment by imstof** [comment]
MOC-R4PAC21U37-S3C is ready to return to service

**Comment by tzumainn** [comment]
> MOC-R4PAC21U37-S3C is ready to return to service

Thanks! Removed it from `hwbroken` and put it on offer.

**Comment by imstof** [comment]
MOC-R4PAC21U25-S1
MOC-R4PAC21U23-S3
MOC-R4PAC21U13-S3
MOC-R4PAC21U11-S1
MOC-R4PAC22U33-S1A
MOC-R4PAC22U31-S1A
MOC-R4PAC22U31-S1C
MOC-R4PAC24U37-S3C
MOC-R4PAC24U31-S1A
are all ready @tzumainn

**Comment by tzumainn** [comment]
@imstof thank you so much!

**Comment by Dhess238** [comment]
MOC-R4PAC08U15-S1
MOC-R4PAC08U25-S3 
are all ready @tzumainn

**Comment by chloetechsquare** [comment]
MOC-R4PAC21U31-S3B is ready @tzumainn

**Comment by tzumainn** [comment]
Thanks! @chloetechsquare I can't seem to clean `MOC-R4PAC21U31-S3B` successfully; it doesn't pxe boot the cleaning image, and just boots from the hard drive. Does cleaning work for you?

**Comment by chloetechsquare** [comment]
Oh shoot, I had it working before I was done for the day on Tuesday. Let me double check it today and I'll get back to you. @tzumainn

**Comment by tzumainn** [comment]
> Oh shoot, I had it working before I was done for the day on Tuesday. Let me double check it today and I'll get back to you. [@tzumainn](https://github.com/tzumainn)

No worries at all! I've experienced similar things.

**Comment by chloetechsquare** [comment]
yup @tzumainn, looks like it's still clean failed, not sure what went wrong after a successful deployment on Tuesday...

**Comment by chloetechsquare** [comment]
MOC-R4PAC21U17-S1
MOC-R4PAC21U33-S3A
MOC-R4PAC21U33-S3C
@tzumainn are ready

**Comment by tzumainn** [comment]
> MOC-R4PAC21U17-S1 MOC-R4PAC21U33-S3A MOC-R4PAC21U33-S3C [@tzumainn](https://github.com/tzumainn) are ready

Verified - thanks!

**Comment by chloetechsquare** [comment]
MOC-R4PAC21U19-S1
MOC-R4PAC22U17-S3
MOC-R4PAC22U15-S1
MOC-R4PAC22U15-S3
@tzumainn are ready :-)

**Comment by tzumainn** [comment]
> MOC-R4PAC21U19-S1 MOC-R4PAC22U17-S3 MOC-R4PAC22U15-S1 MOC-R4PAC22U15-S3 [@tzumainn](https://github.com/tzumainn) are ready :-)

Confirmed - thanks!

**Comment by chloetechsquare** [comment]
MOC-R4PCC04U24 is ready :-) @tzumainn

**Comment by tzumainn** [comment]
confirmed!

**Comment by chloegerhardson** [comment]
MOC-R4PAC22U35-S3B
MOC-R4PAC10U13-S1 is ready :-) @tzumainn

**Comment by tzumainn** [comment]
confirmed - thanks!

**Comment by chloegerhardson** [comment]
MOC-R4PAC08U37-S3C is ready :-) @tzumainn
